modules/my_preferences/templates/form_my_preferences.tpl

```diff
@@ -1,5 +1,5 @@
 <br />
-<form method="post" name="edit_user" id="edit_user">
+<form method="post" name="my_preferences" id="my_preferences" autocomplete="off">
```

Comment:
Why is an ID added to this element?

Reply:
It's not added, it's renamed from `edit_user` to `my_preferences` because those modules are now separate.